src/services/products.ts

```diff
@@ -0,0 +1,12 @@
+import axios from 'axios'
+
+import { ProductResponse } from 'components/pages/Products/Products.types'
+
+const productsAPI = axios.create({
+  baseURL: import.meta.env.VITE_BASE_URL,
+})
+
+export const getProducts = async (page: number): Promise<ProductResponse> => {
+  const response = await productsAPI.get(`api/products?perPage=6&page=${page}`)
```

Comment:
Also perPage should be parametrized and it's props value should be set as default on what we want. In that case `6`, so

`async (page: number, perPage: number)`